RadixWallet/Core/Resources/Resources/en.lproj/Localizable.strings

```diff
@@ -968,14 +967,6 @@ You can always change the guarantee from this default in each transaction.";
 "enterSeedPhrase_titleBabylonMain" = "Enter Main Seed Phrase";
 "enterSeedPhrase_titleBabylon" = "Enter Babylon Seed Phrase";
 "enterSeedPhrase_titleOlympia" = "Enter Olympia Seed Phrase";
-"derivePublicKeys_titleCreateAccount" = "Creating Account";
-"derivePublicKeys_titleCreatePersona" = "Creating Persona";
-"derivePublicKeys_titleImportLegacyAccount" = "Deriving Accounts";
-"derivePublicKeys_titleAccountRecoveryScan" = "Deriving Accounts";
```

Comment:
@matiasbzurovski @kugel3 this change is correct, they should be deleted?

Reply:
I think these have been moved, it's building without these keys